kubernetes/resource_kubernetes_node_taint.go

```diff
@@ -185,10 +200,7 @@ func resourceKubernetesNodeTaintUpdate(ctx context.Context, d *schema.ResourceDa
 	return resourceKubernetesNodeTaintRead(ctx, d, m)
 }
 
-func nodeTaintToId(id string, taints []interface{}) string {
```

Comment:
Hello! Thank you for opening this PR, was wanting to know the reasoning for remove `nodeTaintToId` since we now support multiple taints.